src/themes/ec/_variables.scss

```diff
@@ -4,6 +4,80 @@
 // Use this file to assign default values to component variables
 // Note: if you override css variables it will take over on these values
 
```

Comment:
My idea was to centralize the variables used to customize the components in the theme, so users do not have to update files in each components. All the customization, variable assignment etc could then be kept in a single folder instead of having them spread everywhere. That way the components would be theme agnostic.

Also as there will most probably be some custom themes apart of EC and EU (for EWPP extended), having to create a file in each component would be quite heavy.

I agree that this variables file may grow too much, but if that's the case we could split it in several files (still in the theme folder for me)